setup.py

```diff
@@ -186,7 +186,7 @@ def build_extension(self, ext):
         # cpp/pybind/CMakeList.txt, thus we also set build_lib explicitly,
         # where self.build_lib is the directory for compiled extension modules
         build_temp = self.build_temp
-        self.build_lib = os.path.join(build_temp, 'lib')
+        self.cmake_output_lib = os.path.join(build_temp, 'lib')
         clean_cmake(folders=['_deps', '_exts', self.build_temp])
```

Comment:
Seems not '_ext' anymore

Reply:
Changed to `build_deps`

Reply:
Not commited yet? `clean_cmake(folders=['build_deps', self.build_temp])` works fine.

Reply:
Updated in [18a8979](https://github.com/openxrlab/xrprimer/pull/36/commits/18a8979b073e750778ddfc51e393e9ff5c479705). Sry, only those `_ext` in cmake command are changed yesterday.